### DISCUSSION TITLE
RSS feed for releases

### DISCUSSION THREAD

**Discussion by martinjh99** [jellyfin/jellyfin#8312] [Q&A] [answered]
Hi - I seem to remember that I got an RSS Feed for Jellyfin releases.  I seem to remember it was here on Github I got it from but I can't remember off hand where it was I found it...

Can anyone tell me where it is??

Thanks in advance!

**Comment by nielsvanvelzen** [comment, marked as answer]
https://github.com/jellyfin/jellyfin/releases.atom
